xbmc/games/addons/GameClientInput.h

```diff
@@ -28,9 +28,22 @@ namespace GAME
 {
   class CGameClient;
 
+  /*!
+   * \ingroup games
+   * \brief Handles game controller events for games.
+   *
+   * Listens to game controller events and forwards them to the games (as game_input_event).
+   */
   class CGameClientInput : public JOYSTICK::IInputHandler
   {
   public:
+    /*!
+     * \brief Constructor registers for game controller events at CInputManager.
```

Comment:
game controller events don't pass through CInputManager, they come from the peripherals system. Game clients register with CPortManager, then CPortMapper registers them with g_peripherals. I'm not really happy with the ports code, it's really just an interim until we have a player manager. This will probably change, so keep this description short and vague ;)